docs/index.md

```diff
@@ -17,8 +19,13 @@ integrate it in your use cases, or contribute to its development.
 
 ---
 
-The Dataspace Connector is an IDS connector that is currently being maintained by [sovity](https://sovity.de/).
-The connector was originally developed at the Fraunhofer ISST.
+**This project is currently no longer maintained but looking for new maintainers.**
+
+Applications for maintainers shall be directed to IDSA.
```

Comment:
IDSA is not unambiguous, please refer to International Data Space Association.
Could you please add a contact link for reference, e.g. [info@internationaldataspaces.org](mailto:info@internationaldataspaces.org)

Reply:
done